packages/core/src/rest/ZosmfHeaders.ts

```diff
@@ -166,6 +173,8 @@ export class ZosmfHeaders {
      */
     public static readonly TEXT_PLAIN: IHeaderContent = { "Content-Type": "text/plain" };
 
+    public static readonly APPLICATION_JSON: IHeaderContent = { "Content-Type": "application/json" };
```

Comment:
Do we need to define this option here? It seems like a duplicate of `Headers.APPLICATION_JSON` defined in Imperative.